### PR TITLE
Create sort_array_of_objects.gs

### PR DESCRIPTION
**Pull request by siddhimahawar**

### CODE REVIEW

File: sort_array_of_objects.gs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/**
+* Sort the Array of Objects in Descending order on the label key
+* @param {Object} array will have key value pair
+* @param {String} key will be the label on which sorting has to be done of array
+* @return {Object} sorted Array in descending order 
+*/
+function sortByKey(array, key)
+{
```
Comment (shikhagupta1212) [inline sort_array_of_objects.gs]: move in above line

File: sort_array_of_objects.gs [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+ return array.sort(function(a, b)
+ {
+  var x = a[key]; var y = b[key];
```
Comment (shikhagupta1212) [inline sort_array_of_objects.gs]: var y = b[key]; in next line

File: sort_array_of_objects.gs [hunk 3]
```diff
@@ -0,0 +1,14 @@
+/**
+* Sort the Array of Objects in Descending order on the label key
+* @param {Object} array will have key value pair
+* @param {String} key will be the label on which sorting has to be done of array
+* @return {Object} sorted Array in descending order 
+*/
+function sortByKey(array, key)
+{
+ return array.sort(function(a, b)
```
Comment (shikhagupta1212) [inline sort_array_of_objects.gs]: fix indentation

**Review by shikhagupta1212 — APPROVED** [review]

File: sort_array_of_objects.gs (hunk 1 above)
Comment (siddhimahawar) [reply sort_array_of_objects.gs]: updated

File: sort_array_of_objects.gs (hunk 2 above)
Comment (siddhimahawar) [reply sort_array_of_objects.gs]: Updated

File: sort_array_of_objects.gs (hunk 3 above)
Comment (siddhimahawar) [reply sort_array_of_objects.gs]: updated it

**Review by shikhagupta1212 — APPROVED** [review]

**Pull request merged**